src/test/resources/regressions/features/opaque/opaque-fac1.gobra

```diff
@@ -0,0 +1,27 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package pkg
```

Comment:
I would write a comment in this file saying that this does not exercise the 'opaque' functionality, but serves as a base case to be compared to the tests in file `opaque-fac2.gobra`